hammerspoon/microphone.lua

```diff
@@ -0,0 +1,60 @@
+local message = require('status-message')
+
+local messageMuting = message.new('muted 🎤')
+local messageHot = message.new('hot 🎤')
+local last_mods = {}
```

Comment:
I think the snake_case here comes from the snake_case used in [`control-escape.lua`](https://github.com/adamyonk/keyboard/blob/6d206414ede34f8cbc68b4c0e6a3e2676135da78/hammerspoon/control-escape.lua). Truth be told, `control-escape.lua` only uses snake_case because I haven't gotten around to changing it to the more idiomatic camelCase naming used everywhere else in this repo. 😇 

Would you mind changing these snake_case names to camelCase?